modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.batch-engine-data.json

```diff
@@ -24,15 +24,25 @@
 			"name": "APISort",
 			"objectFields": [
 				{
-					"DBType": "String",
-					"businessType": "Text",
+					"DBType": "Clob",
+					"businessType": "LongText",
 					"externalReferenceCode": "ODATA_SORT",
 					"indexed": true,
 					"indexedAsKeyword": false,
 					"label": {
 						"en_US": "OData Sort"
 					},
 					"name": "oDataSort",
+					"objectFieldSettings":[
+						{
+							"name":"showCounter",
```

Comment:
I think this property is not necessary because it is not going to be displayed in the Frontend part, they will use their own components and no counter will be shown. What do you think?

Reply:
The thing is that, during my testing both yesterday and today, I haven't been able to establish the "maxLength" parameter unless the "showCounter" is true. Maybe it's by design or I can report it.